lib/matplotlib/artist.py

```diff
@@ -1654,5 +1727,9 @@ def kwdoc(artist):
     """
     ai = ArtistInspector(artist)
     return ('\n'.join(ai.pprint_setters_rest(leadingspace=4))
-            if mpl.rcParams['docstring.hardcopy'] else
+            if mpl.rcParams['docstring.hardcopy'] or True else
```

Comment:
This is always `True`  ?  Is this debugging collateral or can we remove this whole conditional?

Reply:
Of course not. I should be more careful not to commit any debug stuff.